Directory.Build.props

```diff
@@ -28,8 +42,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BeforeTargetFiles>$(DlrBuildDir)\Versioning.targets</BeforeTargetFiles>                                                                                                                                                                 
-    <BeforeTargets>GenerateBuildInfo</BeforeTargets>
+    <BeforeTargets></BeforeTargets>
```

Comment:
Do we need this here at all if it is empty?

Reply:
It's not needed, but I left it in there since we're "using" it with
```
<Target Name="BeforeBuildStarts" BeforeTargets="BeforeBuild" DependsOnTargets="$(BeforeTargets)" />
```

We could probably get rid of both since they do nothing...

Reply:
Yeah, let's remove both if we aren't using them.